.env

```
@@ -183,3 +183,17 @@ S3_PUBLIC_ACCESS="true"
+
+# -----------------------------------------------
+# 11. THRESHOLD CONFIGURATION
+# -----------------------------------------------
+NEXT_PUBLIC_INNOVATORS_THRESHOLD=3
+NEXT_PUBLIC_COLLABORATORS_THRESHOLD=3
+NEXT_PUBLIC_FAQ_THRESHOLD=1
+
+# -----------------------------------------------
+# 11. THRESHOLD CONFIGURATION
+# -----------------------------------------------
+NEXT_PUBLIC_INNOVATORS_THRESHOLD=3
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The NEXT_PUBLIC_INNOVATORS_THRESHOLD key is duplicated

(DuplicatedKey)